apps/docs/content/guides/realtime/authorization.mdx

```diff
@@ -0,0 +1,288 @@
+---
+id: 'authorization'
+title: 'Realtime Authorization'
+description: 'Authorization for Supabase Realtime'
+sidebar_label: 'Authorization'
+---
+
+You can control user access to Realtime Channels using RLS policies. By applying RLS, you can:
+
+- Control which users can broadcast to a Channel
+- Control which users can receive broadcasts from a Channel
+- Control which users can publish their presence to a Channel
+- Control which users can receive messages about the presence of other users
+
+<Admonition type="caution">
+
+Realtime Authorization is in public alpha. To use Authorization for your Realtime Channels, use `realtime.js` version `2.10.0` or later.
+
+</Admonition>
+
+## How it works
+
+Realtime uses tables in your database's `realtime` schema to track users and Channel topics. By writing RLS policies against these tables, you can control the access users have to a Channel topic and features within a Channel topic.
+
+The validation is done when the user connects. When their WebSocket connection is established, their permissions are calculated based on:
+
+- The RLS policies on tables in your `realtime` schema
+- The user information sent as part of their [Auth JWT](/docs/guides/auth/jwts)
+- The request headers
+- The Channel topic the user is trying to connect to
+
+<Admonition type="caution">
+
+Increased RLS complexity can impact database performance and connection time, leading to higher connection latency and decreased join rates.
+
+</Admonition>
+
+## Helper functions
+
+You can use the following helper functions when writing RLS policies:
+
+### `realtime.topic`
+
+Returns the Channel topic the user is attempting to connect to.
+
+```sql
+create policy "authenticated users can read all messages in room_1"
```

Comment:
this used to be `"authenticated users can read all messages"`, I changed it to what I _think_ is a more precise description, feel free to change back if I'm wrong